content/en/blog/_posts/Kong-Ingress-Controller-and-Service-Mesh.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: “Kong Ingress Controller and Service Mesh: Setting up Ingress to Istio on Kubernetes”
```

Comment:
```suggestion
title: Kong Ingress Controller and Service Mesh: Setting up Ingress to Istio on Kubernetes
```

Reply:
Because of the colon, the build fails without quotation marks as I discovered with the help of @kbarnard10